AppCenterDistribute/AppCenterDistribute/MSDistribute.m

```diff
@@ -125,10 +125,15 @@ - (void)applyEnabledState:(BOOL)isEnabled {
     [MSAppDelegateForwarder addDelegate:self.appDelegate];
 
     // Enable the distribute info tracker.
-    NSString *distributionGroupId = [MS_USER_DEFAULTS objectForKey:kMSDistributionGroupIdKey];
-    [self.distributeInfoTracker updateDistributionGroupId:distributionGroupId];
     [self.channelGroup addDelegate:self.distributeInfoTracker];
-  } else {
+
+    // Store distributionGroupId in distributeInfoTracker
+    NSString *distributionGroupId = [MS_USER_DEFAULTS objectForKey:kMSDistributionGroupIdKey];
+    if (distributionGroupId) {
```

Comment:
Please format this again. The indentation for `}` seems wrong in this `if` statement.